### DISCUSSION TITLE
offline document

### DISCUSSION THREAD

**Discussion by maobaoqi** [libvips/libvips#3242] [Q&A]
## Questions, enhancements, tips, etc.

Please use libvips discussions for other topics.

https://github.com/libvips/libvips/discussions

## Bug report

**Describe the bug**

**To Reproduce**
1. Use Image '...'
2. Use Configuration '....'
3. See error

**Expected behavior**

**Actual behavior**
A clear and concise description of what actually happen.

**Screenshots**

**Environment**
(please complete the following information)
 - OS: [e.g. Ubuntu 16.04, MacOS 10.13]
 - Vips: [e.g. 8.8, 8.9, etc]

**Additional context**

libvips is a nice image process library and there is offline manual for it? thanks

**Comment by jcupitt** [comment]
Hello @maobaoqi ,

The docs are includes with the sources: 

https://github.com/libvips/libvips/tree/master/doc

If you build libvips yourself, you can get a formatted version of the API docs on the website. Perhaps:

```
cd libvips-x.y.z
meson build --prefix ~/vips -Dgtk_doc=true -Ddoxygen=true
cd build/
ninja install
```

Have a look in `share/gtk-doc/html/libvips` for gtk-doc output, and `share/doc/vips-doc/` for doxygen (the C++ API doc generator) output.
